internal/kafka/internal/workers/kafka_mgrs/kafkas_mgr.go

```diff
@@ -26,6 +26,9 @@ var kafkaMetricsStatuses = []constants2.KafkaStatus{
 	constants2.KafkaRequestStatusDeprovision,
 	constants2.KafkaRequestStatusDeleting,
 	constants2.KafkaRequestStatusFailed,
+	constants2.KafkaRequestStatusSuspended,
+	constants2.KafkaRequestStatusSuspending,
+	constants2.KafkaRequestStatusResuming,
```

Comment:
Might be good to add an integration test for this

Reply:
Added check into existing integration test